### PR TITLE
Ratchets production log to :warn and "paramaterizes" the production l…

### PR DESCRIPTION
**Pull request by cdmo**
…og output file to be named for the current hostname.

#615 

Note: we lose visability on the time it takes for things to render, which is a little concerning to me, but the logs are apparently just too expensive (40GB a day!). Hopefully we'll have datadog APM in place soon and then we won't miss that render time details

### CODE REVIEW

**Comment by whereismyjetpack** [comment]
Is the render time the view render time?

**Comment by cdmo** [comment]
Yes. And that graph looks 🔥 

Looking forward to that contract finalizing or whatever. Thanks.

**Comment by banukutlu** [comment]
Also leaving this here https://github.com/roidrage/lograge just to note

**Comment by whereismyjetpack** [comment]
In ScholarSphere, we are using lograge, with a custom JSON formatter to emit logs into a machine readable format. 

https://github.com/psu-stewardship/scholarsphere-4/pull/240

**Comment by cdmo** [comment]
Ok. I'm proposing we use `lograge` and set level to `:info`. `:info` gives us far less verbose logs *but still gives us some insight into the performance of requests*. Basically each http request on the application gets an executive summary.

`:debug` in lograge, to me, isn't great, so I don't want to use it locally. Rails default logger gives better details, specifically the render time of each discrete partial.

**Comment by whereismyjetpack** [comment]
For local development we flip lograge off to get the goods locally. LGTM

File: config/environments/production.rb [hunk 1]
```diff
@@ -81,13 +81,26 @@
...
     logger.formatter = config.log_formatter
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  # Adds lograge and parametrizes log name. Capture parameters other than `controller` and `action` in lograge. ALso
+  # captures timestamp.
+  config.lograge.enabled = true
+  config.lograge.custom_options = lambda do |event|
+    params = event.payload[:params].reject { |k| %w(controller action).include?(k) }
+    {
```
Comment (banukutlu) [inline config/environments/production.rb:95]: thanks @cdmo!

**Pull request merged**